src/System.Text.Json/pkg/Microsoft.Bcl.Json.Sources.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  <ItemGroup Condition="'$(ExcludeEmbeddedResourceFromPackage)' != 'true'">
+    <_embeddedResourceFiles Include="$(MSBuildThisFileDirectory)*.resx" />
+    <EmbeddedResource Include="@(_embeddedResourceFiles -> '%(FullPath)')">
+      <ManifestResourceName>$([System.String]::Copy('%(FileName)').Replace('.neutral',''))</ManifestResourceName>
```

Comment:
I'm not a huge fan of this `neutral` convention and the wildcard.  I think hardcoding would be a lesser evil.  Alternatively if you wanted to get fancy you could generate it in the source CSProj.